tools/apiview/parsers/swagger-api-parser/SwaggerApiParser/Program.cs

```diff
@@ -114,6 +118,13 @@ static async Task HandleGenerateCodeFile(IEnumerable<string> swaggers, string ou
 
             var codeFile = root.GenerateCodeFile();
             var outputFilePath = Path.GetFullPath(output);
+
+            if (useTagForOutput)
+            {
+                output = $"{readmeTag}.json";
```

Comment:
Where do you rename this with new or baseline postfix? how is it handled when both source and target has same tag?

Reply:
https://github.com/Azure/azure-rest-api-specs/blob/00d50cba07376c494f3b50647b847cd4ba348a04/eng/scripts/Create-APIView.ps1#L118